tests/unit/query_helpers/test_query_helper.py

```diff
@@ -0,0 +1,41 @@
+"""Unit tests for query helper class."""
+
+from ols.src.query_helpers import QueryHelper, camel_to_snake
+from ols.utils import config
+
+
+def test_camel_to_snake():
+    """Test camel case to snake case conversion."""
+    assert camel_to_snake("") == ""
+    assert camel_to_snake("dog") == "dog"
+    assert camel_to_snake("LOUD") == "loud"
+    assert camel_to_snake("quiet") == "quiet"
+    assert camel_to_snake("OCP4") == "ocp4"
+    assert camel_to_snake("Is there a space?") == "is there a space?"
+    assert camel_to_snake("rainyDay") == "rainy_day"
+    assert camel_to_snake("NiceDay") == "nice_day"
+    assert camel_to_snake("AGoodDay") == "a_good_day"
+
+
+class TestQueryHelper:
+    """Test the query helper class."""
+
+    def test_defaults_used(self):
+        """Test that the defaults are used when no inputs are provided."""
+        # TODO: I don't like this setting - should be possible to
+        # refactor after OLS-89
+        config.load_config_from_env()
```

Comment:
this will need to be fixed now that the config yaml change has landed

```
FAILED tests/unit/query_helpers/test_query_helper.py::TestQueryHelper::test_defaults_used - AttributeError: module 'ols.utils.config' has no attribute 'load_config_from_env'
```